pkg/cluster/inventory.go

```diff
@@ -85,7 +85,7 @@ func (i *DefaultInventory) createCluster(contractVersion int64, cluster *keb.Clu
 		Cluster:    cluster.Cluster,
 		Runtime:    string(runtime),
 		Metadata:   string(metadata),
-		Kubeconfig: "fixme", //TODO: use correct model field as soon as kubeconfig is provided
+		Kubeconfig: cluster.Kubeconfig, //TODO: use correct model field as soon as kubeconfig is provided
```

Comment:
Do we still need the TODO comment?

Reply:
Currently, we haven't yet agreed with Gophers where the kubeconfig will be located in the JSON. I was just adding it now to the top level to get the end2end scenario working.

We have also issue #44 open for aligning it - I would leave it for now and drop it as part of this issue (will add a comment in the issue ticket to drop this TODO)